radix-engine-tests/tests/metering.rs

```diff
@@ -35,6 +35,7 @@ fn execute_with_time_logging(
 }
 
 #[test]
+#[ignore]
```

Comment:
Accidental change?  You might find `cargo nextest run --no-fail-fast` useful. :)